pages/common/7z.md

```diff
@@ -10,6 +10,10 @@
 
 `7z x {{archived}}`
 
+- Extract an archive with user-defined output path:
+
+`7z x {{archived}} -o{{path/to/output}}`
```

Comment:
Shouldn't there be a space between "o" and "{" ?

Reply:
I thought that too at first, but it looks like it doesn't have a space according to the docs/man page.

Reply:
Nope, it doesn't have space between -o and {}.

Reference: http://7zip.bugaco.com/7zip/MANUAL/switches/output_dir.htm

Reply:
Cool.